vocabularies/Common.xml

```diff
@@ -707,6 +707,16 @@ The minimum structure is
 
       <Term Name="ValueListWithFixedValues" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">
         <Annotation Term="Core.Description" String="If specified as true, there's only one value list mapping and its value list consists of a small number of fixed values" />
+        <Annotation Term="Validation.ApplicableTerms">
+          <Collection>
+            <String>Common.ValueListShowValues</String>
+          </Collection>
+        </Annotation>
+      </Term>
+
+      <Term Name="ValueListShowValues" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Annotation">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="A value list with a very small number of fixed values, can decide to show all values immediately" />
```

Comment:
```suggestion
        </Term>
        <Term Name="ValueListWithImmediateValues" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">
          <Annotation Term="Core.Description" String="If specified as true, there's only one value list mapping and its value list consists of a small number of fixed values that can be shown immediately" />
```

Reply:
Alternatively, can't you use `Common.ValueListMapping/FetchValues = 1` for that?

Reply:
Hi Heiko, 
thank you very much for your proposals - really appreciate that! First I would like to comment on the second one (using FetchValues), which I immediately liked, but then I thought about apps/customers that used that already, and afterwards set the _Common.ValueListWithFixedValues_ annotation. Today they see in Fiori Elements a dropdown listbox. But after FE would use that to show e.g. a group of radio buttons in this constellation they would see radio buttons (independent if the amount of values really fits this visualization) - I think they will regard this as a regression. 
Wrt the other proposal I personally would make this term also only available as annotation of _Common.ValueListWithFixedValues_. I see the new term as a kind of "specialization" of this existing term - it's a clear subset.
But as said these are my personal opinions - maybe @ralfhandl or @stefan0816 want to comment on that.

Reply:
Thanks, Ralf! So, if Heiko doesn't veto, we stick to the original term definition, but we might discuss the name of the new term: We have "ValueListShowValues", "ValueListWithImmediateValues", and "ValueListShowValuesImmediately" - the last one seems to be the most expressive, but it's also the longest. So, what's your choice?

Reply:
I like expressive and prefer `ValueListShowValuesImmediately`.